### ISSUE TITLE
Don't block on indexer when sending transactions

### ISSUE THREAD

**Issue by vgrichina** [near/near-wallet#1747]
## Story

As a USER who experiences connectivity problems or total downtime of blockchain indexer, I want to still access wallet if I already have keys, so I can send NEAR tokens and perform other transactions.

## Acceptance Criteria

- [ ] Send Money UI works (regardless of whether staking balances are loaded)
- [ ] Login into app works
- [ ] Signing transactions from app works
- [ ] Managing recovery methods works

## Unhappy Path

If contract helper is completely down – some of operations (like stuff related to recovery keys) might still get blocked.

## NOTES

Basically the goal is to make sure that as long as NEAR RPC is up - user can send basic transactions in wallet.

Likely the easiest way to achieve this is to load available balance separately from staking / total balances and only wait for available balance to be loaded before allowing to send transaction.
